jsk_data/src/jsk_data/download_data.py

```diff
@@ -60,11 +60,13 @@ def extract_file(path, to_directory='.', chmod=True):
     return root_files
 
 
-def decompress_rosbag(path, quiet=False, chmod=True):
+def decompress_rosbag(path, quiet=False, chmod=True, force=True):
```

Comment:
This is just a comment, but why is the default value of argument `force` different from https://github.com/jsk-ros-pkg/jsk_common/pull/1621/files#diff-47c43621e14db9c2464d84a6e6e6ee81R129 ?

Reply:
we should not change default behaviour, so i'll change it `force=False` in default.